lectures/long_run_growth.md

```diff
@@ -73,75 +72,71 @@ import numpy as np
 from collections import namedtuple
 ```
 
-
 ## Setting up
 
 A project initiated by [Angus Maddison](https://en.wikipedia.org/wiki/Angus_Maddison) has collected many historical time series related to economic growth,
 some dating back to the first century.
 
-The data can be downloaded from the [Maddison Historical Statistics webpage](https://www.rug.nl/ggdc/historicaldevelopment/maddison/) by clicking on the "Latest Maddison Project Release". 
+The data can be downloaded from the [Maddison Historical Statistics](https://www.rug.nl/ggdc/historicaldevelopment/maddison/) by clicking on the "Latest Maddison Project Release". 
 
-For convenience, here is a copy of the {download}`2020 data <datasets/mpd2020.xlsx>` in `Excel` format.
+We are going to read the data from a QuantEcon GitHub repository.
 
-Let's read it into a pandas dataframe:
+Our objective in this section is to produce a convenient `DataFrame` instance that contains per capita GDP for different countries.
+
+Here we read the Maddison data into a pandas `DataFrame`:
 
 ```{code-cell} ipython3
-data = pd.read_excel("datasets/mpd2020.xlsx", sheet_name='Full data')
-data
+data_url = "https://github.com/QuantEcon/lecture-python-intro/raw/main/lectures/datasets/mpd2020.xlsx"
+data = pd.read_excel(data_url, 
+                     sheet_name='Full data')
+data.head()
 ```
 
-
-We can see that this dataset contains GDP per capita (gdppc) and population (pop) for many countries and years.
+We can see that this dataset contains GDP per capita (`gdppc`) and population (pop) for many countries and years.
 
 Let's look at how many and which countries are available in this dataset
 
 ```{code-cell} ipython3
-len(data.country.unique())
+countries = data.country.unique()
+len(countries)
 ```
 
-
 We can now explore some of the 169 countries that are available. 
 
 Let's loop over each country to understand which years are available for each country
 
 ```{code-cell} ipython3
-cntry_years = []
-for cntry in data.country.unique():
-    cy_data = data[data.country == cntry]['year']
+country_years = []
+for country in countries:
+    cy_data = data[data.country == country]['year']
     ymin, ymax = cy_data.min(), cy_data.max()
-    cntry_years.append((cntry, ymin, ymax))
-cntry_years = pd.DataFrame(cntry_years,
-                           columns=['country', 'Min Year', 'Max Year']).set_index('country')
-cntry_years
+    country_years.append((country, ymin, ymax))
+country_years = pd.DataFrame(country_years,
+                    columns=['country', 'min_year', 'max_year']).set_index('country')
```

Comment:
good variable names `min_year` as they are now accessible as attributes on the dataframe. 👍

Reply:
Let's add this as a `pandas` style guide tip.